src/site/stages/build/drupal/home.js

```diff
@@ -17,10 +17,28 @@ function addHomeContent(contentData, files) {
       },
     } = contentData;
 
+    // Liquid does not have a good modulo operator, so we let the template know when to end a row.
+    const hubs = homePageHubListQuery.itemsOfEntitySubqueueHomePageHubList.map(
+      (hub, i) => {
+        // We want 3 cards per row.
+        if ((i + 1) % 3 === 0) {
+          hub = {
+            ...hub,
+            endRow: true,
+          };
+        }
+        return hub;
+      },
+    );
```

Comment:
Liquid does have a [modulo filter](https://shopify.github.io/liquid/filters/modulo/)

Reply:
Not sure if that would really be cleaner or not though 🤷‍♂

Reply:
:thinking: I think I'd prefer to see it in the liquid template, but if using that modulo gets too wonky, this is fine too.

Reply:
Ahhh, you beat me to it. Sounds good to me, @jonbot :+1: